src/commands.rs

```diff
@@ -3477,18 +3481,22 @@ fn cmd_edit(ui: &mut Ui, command: &CommandHelper, args: &EditArgs) -> Result<(),
 
 fn cmd_new(ui: &mut Ui, command: &CommandHelper, args: &NewArgs) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
-    let parent = workspace_command.resolve_single_rev(&args.revision)?;
-    let commit_builder = CommitBuilder::for_open_commit(
-        ui.settings(),
-        parent.id().clone(),
-        parent.tree_id().clone(),
-    )
-    .set_description(args.message.clone());
+    let mut commits = vec![];
+    let mut parent_ids = vec![];
+    for revision_arg in &args.revisions {
```

Comment:
maybe better to assert that the revisions isn't empty. If it were empty, new commit would be created on top of the root, which seemed very wrong.

Reply:
The default for the arguments is set to `@`, so I don't think clap ever lets this be an empty list, right?

Reply:
Oh, you're just saying we should assert that my assumption about clap holds here. Makes sense. I'll add that assertion.

Reply:
That's now done.